includes/Services/FlowService.php

```diff
@@ -0,0 +1,203 @@
+<?php
+namespace NewfoldLabs\WP\Module\Onboarding\Services;
+
+use NewfoldLabs\WP\Module\Onboarding\Data\Options;
+use NewfoldLabs\WP\Module\Onboarding\Data\Flows;
+use NewfoldLabs\WP\Module\Onboarding\Data\Data;
+
+class FlowService {
+
+	public static function initalize_flow_data() { 
```

Comment:
`Note for Later`: From now on Github actions will run a lint workflow on all our PHP files which also checks for docblocks, you can use https://marketplace.visualstudio.com/items?itemName=neilbrayfield.php-docblocker or https://marketplace.visualstudio.com/items?itemName=ronvanderheijden.phpdoc-generator to generate these. We can skip for now but lets follow up in flow data parts 2 and 3